### PR TITLE
env default url added for custom theming

### PR DESCRIPTION
**Pull request by Ajay-aot**
# Issue Tracking

JIRA:  https://aottech.atlassian.net/browse/FWF-2184

# Changes
env default url added for custom theming

# Screenshots (if applicable)
Screenshot (208)

### CODE REVIEW

**Review by Josephalexantony-aot — APPROVED** [review]

**Review by AbijithS-aot — APPROVED** [review]

File: forms-flow-web-root-config/sample.env [hunk 1]
```diff
@@ -87,16 +87,18 @@ APPLICATION_NAME=formsflow.ai
...
+#the json data should be below format.
+
 #  `{
 #  "--navbar-background": "blue",
 #  "--navbar-items": "grey",
 #  "--navbar-active": "white"
 #  }`
 
-#CUSTOM_THEME_URL=
+#CUSTOM_THEME_URL=/themeConfig/customTheme.json
```
Comment (shuhaib-aot) [inline forms-flow-web-root-config/sample.env:101]: I have one doubt. how users can get the variable names. and please add more description on sample.env to understand how they can override the theme

**Review by shuhaib-aot — APPROVED** [review]

File: forms-flow-web-root-config/sample.env (hunk 1 above)
Comment (Ajay-aot) [reply forms-flow-web-root-config/sample.env:101]: okay

**Review by abhilash-aot — APPROVED** [review]

File: forms-flow-web-root-config/sample.env (hunk 1 above)
Comment (abhilash-aot) [reply forms-flow-web-root-config/sample.env:101]: There is some description added for sample.env now , later will add do documentation

Comment (abhilash-aot) [reply forms-flow-web-root-config/sample.env:101]: image

**Pull request merged**